frameworks/Swift/vapor/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# install swiftenv for managing Swift versions
+eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/02090c7ede5a637b76e6df1710e83cd0bbe7dcdf/swiftenv-install.sh)"
```

Comment:
That's strange. The script is specifically for CIs like Travis so maybe it won't work on your local environment. The other option is installing `swiftenv` manually: https://github.com/kylef/swiftenv